kustomize/manager/kustomization.yaml

```diff
@@ -8,4 +8,4 @@ configurations:
 images:
   - name: controller
     newName: ghcr.io/kubecfg/kubit
-    newTag: v0.0.15@sha256:e36f6029f6c55a8cc484f032a30a1ec601179c9656797d0fa14ff60da7942cbe
+    newTag: v0.0.15
```

Comment:
This comes from here https://github.com/kubecfg/kubit/pull/400

I incorrectly created the tag first, but I forgot that we have automation which handles using SHAs etc. Once the _real_ tag is used, this will go back to an `@sha256` for the image from the renovate bot.